sdk/core/azure-core/src/test/java/com/azure/core/util/logging/ClientLoggerTests.java

```diff
@@ -105,13 +108,17 @@ public void logAtUnsupportedLevel(int logLevel) {
     @ValueSource(ints = { 1, 2, 3, 4 })
     public void logWhenLoggingDisabled(int logLevel) {
```

Comment:
Should rename this test

Reply:
change to `logWhenLoggingInvalidNumeric`